src/app/views/query-runner/request/util.ts/CssHelpers.ts

```diff
@@ -0,0 +1,14 @@
+// DetailsList has a wrapper div that is added at runtime with the classname
```

Comment:
You could try adding the height property on the .scss file to achieve the same without having to add this function

Reply:
Also, I think it's worth investigating the parent heights... There may be some wrong configurations set